doc/source/whats_new.rst

```diff
@@ -31,6 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
```

Comment:
as you modified the mixin you added the feature to all objects (Epochs, Evoked etc...). Although only Raw is tested.